script/model.py

```diff
@@ -571,9 +578,10 @@ def calculate_loss(self, t1, t2):
             + loss_kl_belief_transition_z_time2_layer1
             + reconstruction_loss
         )
+        
         loss = torch.mean(loss)
 
-        return loss
+        return loss, z_time1_smoothing, z_time2
```

Comment:
This seems to add pretty different behavior. Did you confirm that the model still trains? (e.g. I didn't see corresponding updates to calls to this function)